.github/workflows/npm-publish.yml

```
@@ -0,0 +1,21 @@
+# This workflow will publish a package to npmjs.com when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on: push
+
+jobs:
+  publish-npm:
+    if: github.ref == 'refs/heads/master' && github.event_name == 'push' && contains(github.event.head_commit.message, 'chore(release)')
```

My understanding is that `chore(release)` must be the first thing in the commit message in this scenario, correct?

If yes, I would consider replacing:
```
contains(github.event.head_commit.message, 'chore(release)')
```
with https://docs.github.com/en/actions/learn-github-actions/expressions#startswith, just to make sure that if we ever encounter `chore(release)` in the middle of the commit message, we don't do an accidental release.

---

It's not a problem to change it. It never happened that it was included in the commit message.

```suggestion
    if: github.ref == 'refs/heads/master' && github.event_name == 'push' && startsWith(github.event.head_commit.message, 'chore(release)')
```

---

I'm leaving the decision to you, then 🙂